spot-client/src/common/app-state/remote-control-service/selectors.js

```diff
@@ -16,6 +16,20 @@ export function getOptimisticAudioMuteState(state) {
         : undefined;
 }
 
+/**
+ * A selector which returns the desired raise hand state of any pending raise
+ * hand request.
+ *
+ * @param {Object} state - The Redux state.
+ * @returns {boolean|undefined} The boolean for the pending raise hand state
+ * will be returned if a request is pending, otherwise undefined is returned.
+ */
+export function getOptimisticHandRaisedState(state) {
```

Comment:
Is this necessary in order to raise your hand while you haven't joined yet? If so, how about we disable button until then?